src/migrations/20210106160931-create-activity-reports.js

```diff
@@ -47,10 +47,6 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING,
       },
-      programTypes: {
```

Comment:
Wouldn't best practice be to write a new migration rather than edit this one?

Reply:
I don't think we are removing the database column. I almost wonder if I shouldn't change the migrations at all.

Reply:
yeah thats a good point. so would it have to stay here and where its changed on the model?

Reply:
Yeah once a migration hits main it needs to not change. Any modifications to the schema need to happen with a new migration.